doc/index.rst

```diff
@@ -0,0 +1,366 @@
+the image is displayed, while the transparency may be changed at any time.
+
+.. code-block:: py
+    :name: mask-manipulation
+
+    display1.setMaskPlaneColor('DETECTED', afw_display.GREEN)
+    display1.mtv(calexp)
+    display1.setMaskTransparency(30)
+
+Image scale, zoom, pan
+----------------------
+
+The ``display1`` object includes methods for setting the image scale or
+stretch, the zoom and the pan position.
+
+.. code-block:: py
+    :name: scale-zoom-pan
+
+    display1.scale('log', -1, 10, 'sigma')
+    display1.zoom(4)
+    display1.pan(1064, 890)
+
+Overlaying symbols
+------------------
+
+The :meth:`display1.dot` method will overlay a symbol at a point.
+
+.. code-block:: py
+    :name: ff-dot
+
+    display1.dot('x', 1064, 890, size=8, ctype=afw_display.RED)
+
+interact() method not implemented
+---------------------------------
+
+The :meth:`display1.interact()` method is not implemented, due mainly to
+limitations of a browser window recognizing a keypress event.
+
+
+.. _lsst-display-firefly-installing:
+
+Installing lsst.display.firefly
+===============================
+
+Since `display_firefly` is not yet included in the `lsst_distrib` set of stack
+packages, this section outlines several installation scenarios.
+
+.. _lsst-display_firefly-eups-distrib-install:
+
+Installing with eups distrib install
+------------------------------------
+
+To check for published distributions of `display_firefly`, use
+
+.. code-block:: shell
+    :name: eups-distrib-list
+
+    eups distrib list display_firefly -s https://sw.lsstcorp.org/eupspkg
+
+This command will return the published versions of `display_firefly`, with
+the third item displayed as the EUPS version. Provide that version to
+`eups distrib install display_firefly`, e.g.:
+
+.. code-block:: shell
+    :name: eups-distrib-install
+
+    eups distrib install display_firefly master-g05bec400a4
+
+The version may not be compatible with the version of the stack you are
+using. This will be fixed when `display_firefly` is included in the
+`lsst_distrib` distribution.
+
+
+Developer installation from source code
+---------------------------------------
+
+Install using lsstsw
+^^^^^^^^^^^^^^^^^^^^
+
+If using `lsstsw` to develop with the stack, rebuild the package:
+
+.. code-block:: shell
+    :name: rebuild-firefly
+
+    rebuild display_firefly
+
+Note the build number NNNN that is output by the rebuild process, then:
+
+.. code-block:: shell
+    :name: tag-clone
+
+    eups tags --clone bNNNN current
+
+Install using Github and eups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set up the stack with `lsst_distrib` or an obs package such as `obs_sdss` -- just
+use something that already contains `lsst.afw`.
+
+Clone and install `display_firefly`:
+
+.. code-block:: shell
+    :name: git-display-ff
+
+    git clone https://github.com/lsst/display_firefly
+    cd display_firefly
+    setup -j -r . -t $USER
+    scons -Q -j 6 opt=3
```

Comment:
FWIW, I don't think you should suggest a number of cores to use: e.g. my laptop only has 4 cores.  `scons` is pretty good at figuring out how many cores to use.

I'd also remove the opt=3 option.  This is a pure python package so is a no-op, I think.